### ISSUE TITLE
Travel timer includes time between sessions in calculations

### ISSUE THREAD

**Issue by CMDRDevildoll** [EDDiscovery/EDDiscovery#2417]
I have noticed that if you have a travel timer set across multiple game sessions, or initiate one at the start of a game session that is in some close amount of time from a previous session, the timer includes that gap in the travel calculations/statistics.

Picture one shows the previous session
![1](https://user-images.githubusercontent.com/47367900/52303046-42d79e80-298f-11e9-8708-106106a92211.jpg)
Picture two shows the new timer at the start of a new game session about 20 minutes later, that already on the first second of the timer has 19:50 clocked on it, which also happens to be the gap between the start timer and the entry below it.
![2](https://user-images.githubusercontent.com/47367900/52303050-44a16200-298f-11e9-9028-3144bb4067c1.jpg)

Picture 3 shows what happens if start the timer one line higher, it instead starts at 0:35, which is again the gap to the previous entry.
![3](https://user-images.githubusercontent.com/47367900/52303053-466b2580-298f-11e9-9802-b2c21513b4a5.jpg)

So to summarize, travel timer sometimes includes duration to the entry before it, which is an issue both in the case of a travel timer starting at the beginning of a new game session, but also in the case where a timer spans across multiple game sessions with downtime in between them.

**Comment by robbyxp1** [comment]
I'll have a look.  This bit has not had much TLC since the shutdown event came in..

**Comment by robbyxp1** [comment]
Reengineered - copes with shutdown and with previous logs which did not have a shutdown.  If fileheader is more than 30 mins past the previous entry its not counted ..

thanks for the report. will be in the next version.

**Comment by CMDRDevildoll** [comment]
Thanks a lot!
